api/src/main/java/io/strimzi/api/kafka/model/kafka/KafkaStatus.java

```diff
@@ -58,6 +59,8 @@ public void setKafkaNodePools(List<UsedNodePoolStatus> kafkaNodePools) {
         this.kafkaNodePools = kafkaNodePools;
     }
 
+    @Deprecated
+    @DeprecatedProperty(description = "The `registeredNodeIds` property is deprecated and it is not used anymore. It will be removed in the future.")
```

Comment:
You should add here also the `@PresentInVersions("v1beta2")` annotation I guess to make sure it does not propagate to `v1`.

Reply:
Good point. Fixed.